ouroboros-network/src/Ouroboros/Network/NodeToNode.hs

```diff
@@ -165,7 +165,8 @@ nodeToNodeProtocols chainSync
     -- TODO: these limits depends on various parameters instantiated by
     -- 'Ouroboros.Consensus.Node'.  We should parametrize over them, but these
```

Comment:
parametrize -> parameterize (or -....se - I'm UK,...)

Reply:
o really, in UK its spelled this way, I wouldn't think.

Reply:
This comment needs to be removed.  I fixed it in a later patch.